rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/data/JobInfoImpl.java

```diff
@@ -25,9 +25,7 @@
  */
 package org.ow2.proactive.scheduler.rest.data;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Are you sure you need all utils here ? You just replaced all the three (List, AL & Map) with just two (Set & Hashset) ?

Reply:
Yes I'm sure :), thanks for the review